modules/op-release-notes-1-8.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+:_content-type: REFERENCE
+[id="op-release-notes-1-8_{context}"]
+= Release notes for {pipelines-title} General Availability 1.8
+
+With this update, {pipelines-title} General Availability (GA) 1.8 is available on {product-title} 4.9, 4.10, and 4.11.
```

Comment:
1.8 is shipped (and supported) on 4.10, 4.11, 4.12 (see https://spaces.redhat.com/pages/viewpage.action?spaceKey=SRVKP&title=Pipeline+Component+Matrix)

Reply:
Thanks for the link, @vdemeester. Because 4.11 and 4.12 won't be available when 1.8 GA goes live, we must state this as follows:
```suggestion
With this update, {pipelines-title} General Availability (GA) 1.8 is available on {product-title} 4.10, and is planned to be available on {product-title} 4.11 and 4.12.
```

Reply:
yes 👍🏼